pytest_ibutsu.py

```diff
@@ -42,6 +44,14 @@
 MAX_CALL_RETRIES = 3
 
 
+class DateTimeEncoder(JSONEncoder):
+    """ Handle datetime objects in the archiver. """
+
+    def default(self, obj):
+        if isinstance(obj, (date, datetime)):
+            return obj.isoformat()
+
```

Comment:
There's no else clause here or inherited method call, but I'm not very familiar with overriding JSON encoders, so I'm presuming this will just work?

Reply:
Yup, I pulled this from https://pynative.com/python-serialize-datetime-into-json/

Tested locally both with the backend off at the start of the test run and turning it off during the test run and the pytest session continued as expected. Also tested with the backend on to make sure it doesn't interfere with any data.